tailnet/conn.go

```diff
@@ -467,10 +483,7 @@ func (c *Conn) UpdateNodes(nodes []*Node, replacePeers bool) error {
 			Endpoints:  node.Endpoints,
 			DERP:       fmt.Sprintf("%s:%d", tailcfg.DerpMagicIP, node.PreferredDERP),
 			Hostinfo:   hostinfo.New().View(),
-			// Starting KeepAlive messages at the initialization
-			// of a connection cause it to hang for an unknown
-			// reason. TODO: @kylecarbs debug this!
-			KeepAlive: ok && peerStatus.Active,
+			KeepAlive:  true,
```

Comment:
A note for reviewers: I think connections were hanging because of the Tailscale "trimming" of the WG nodes, which I have disabled in this PR.

Reply:
I wonder if this could be the underlying cause of the test failures? I'm seeing a lot of changes that I wouldn't expect to be needed, so thought I'd point it out.

Reply:
Turning this on is what triggers wireguard to go and proactively attempt to connect to a peer, rather than just passively waiting for the peer to connect.

I am seeing some complaints in the test about peers getting "invalid response messages".  I can't help but wonder if it's related to each side attempting to initiate a handshake at the same time.  Maybe that's delaying the handshakes by 5 seconds a lot more.

I could make it so that only the clients turn this on...

Reply:
https://github.com/coder/coder/pull/8492/commits/07df3abae938ac4341115e306e49385ed9956323

I'm out of time before my vacation.  We can leave this until I get back, or someone else can take over